### DISCUSSION TITLE
about github account

### DISCUSSION THREAD

**Discussion by sagar0710** [community/community#62800] [Repositories]
### Select Topic Area

Question

### Body

Is it possible that I don't upload my portrait to my account in github?

**Comment by goldest-star** [comment]
Yes, it is possible to have a GitHub account without uploading a portrait or profile picture. GitHub allows users to customize their profiles by adding a profile picture, but it is not mandatory. You can choose to leave your profile picture blank or use a default avatar provided by GitHub.

**Reply by sagar0710** [reply]
good, thanks.

**Comment by queenofcorgis** [comment]
@sagar0710 @goldest-star GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
